setup.py

```diff
@@ -91,19 +91,13 @@ def get_version(version_file):
         "cyhy-logging @ git+https://github.com/cisagov/cyhy-logging.git@first-commits",
         "jsonschema",
         "rich",
-        "setuptools >= 24.2.0",
+        "setuptools",
```

Comment:
Why did you remove this pin?  Should we be removing it everywhere?

Reply:
Feel free to do that.  There was no comment.  So, for consistency, I proactively removed it for you.  😗 

There is a commit somewhere, where I discussed this.  (different repo).  I'll see if I can track that down for you.

Reply:
Tracked it down over in `cyhy-config`.  Explanation here:
- [8df2e692d2920743c40368a496ca154769b2985a](https://github.com/cisagov/cyhy-config/commit/8df2e692d2920743c40368a496ca154769b2985a)

Reply:
> Feel free to do that. There was no comment. So, for consistency, I proactively removed it for you. 😗

Touché!

Reply:
See cisagov/skeleton-python-library#142.